.github/workflows/build.yml

```diff
@@ -99,26 +111,34 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python3: ${{fromJson(needs.common_config.outputs.linux_python_versions)}}
-        python_deps: [""]
+        python3: ${{fromJson(vars.LINUX_PYTHON_VERSIONS || '[6, 7, 8, 9, 10, 11]')}}
```

Comment:
Can we get rid of `LINUX_PYTHON_VERSIONS`, don't see why we need that additional degree of freedom?

Reply:
I use it on my fork to reduce the number of jobs.